src/Presentation/ConfigEditPageTextBuilder.php

```diff
@@ -124,4 +49,31 @@ private function getExampleContents(): string {
 		return $example;
 	}
 
+	private function getItemTypesData(): array {
+		$itemTypes = $this->config->getItemTypes();
+		if ( $itemTypes === [] ) {
+			return [];
+		}
+
+		$data = [];
+		foreach ( $itemTypes as $itemType ) {
+			$id = $itemType->getSerialization();
+			if ( !$id ) {
```

Comment:
This check should be explicit. However, why do we need to check for this? The `ItemId` constructor already checks that the format is valid. So the `ItemId`s you get from the config are already valid.